src/transformers/generation/flax_utils.py

```diff
@@ -347,7 +347,6 @@ def generate(
             eos_token_id = generation_config.eos_token_id
             if isinstance(eos_token_id, list):
                 eos_token_id = eos_token_id[0]
-            logger.warning(f"Setting `pad_token_id` to `eos_token_id`:{eos_token_id} for open-end generation.")
```

Comment:
This is a legacy warning (before I joined 👀 ) that 
a) is noisy 
b) doesn't affect generation other than the ability to try to infer the attention mask when it is not passed

In the specific case of assisted generation, it was emitted once per assistant model call, so many times 😅 More harmful than useful.

Reply:
Strong agree on removing this one!